src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -121,7 +121,10 @@ export const TopBarEquipmentSearchDialog: FunctionComponent<
     return (
         <ElementSearchDialog
             open={isDialogSearchOpen}
-            showResults={equipmentsFound.length > 0 || isLoading}
+            showResults={
```

Comment:
I think it's better to change commons-ui code here, but it's a choice...

Reply:
It can be done in both but since the PO will only be available tomorrow I prefer to handle it in gridsuite.